src/locale/locales/de/messages.po

```diff
@@ -3266,27 +3266,19 @@ msgstr "Sicherheitsschritt erforderlich"
 
 #: src/components/TagMenu/index.web.tsx:66
 msgid "See {truncatedTag} posts"
-msgstr ""
+msgstr "Siehe {truncatedTag}-Beiträge"
 
 #: src/components/TagMenu/index.web.tsx:83
 msgid "See {truncatedTag} posts by user"
-msgstr ""
+msgstr "Siehe {truncatedTag}-Beiträge von diesem Benutzer"
```

Comment:
```suggestion
msgstr "Siehe {truncatedTag}-Beiträge des Benutzers"
```

`Beiträge von diesem Benutzer` is `posts by this user` but since there is no **this** in the English locale I'd leave it out in German as well